app/controllers/records_controller.rb

```diff
@@ -0,0 +1,35 @@
+class RecordsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @group = Group.includes(:records).find(params[:group_id])
+    @records = @group.records.order(created_at: :desc)
+  end
+
+  def new
+    @group = Group.find(params[:group_id])
+    @record = @group.records.build
+  end
+
+  def create
+    @group = Group.find(params[:group_id])
+    @record = @group.records.build(record_params)
+    @record.author = current_user
+
+    Rails.logger.info @record.inspect
```

Comment:
- [ ] Kindly make sure that all groups and transactions are related only the user that created them , currently all users share and manage the same groups and taxes, if i create an account and then create a group , i can see & manage that same group from another account, kindly fix that for transactions & groups by using current_user provided by devise 🙏